### PR TITLE
add oncoThreads preprint

### PR DESCRIPTION
**Pull request by wangqianwen0418**

### CODE REVIEW

File: _publications/harbig-2020-osf.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+year: 2020
+type: preprint
+
+publisher: "https://osf.io/b3n4u/"
+cite:
+  authors: "Theresa Harbig, Sabrina Nusrat, Tali Mazor, Qianwen Wang, Alexander Thomson, Hans Bitter, Ethan Cerami, Nils Gehlenborg"
+  published: "OSF Preprints b3n4u"
+---
+<b>Motivation:</b> Molecular profiling of patient tumors and liquid biopsies over time with next-generation sequencing technologies and new immuno-profile assays are becoming part of standard research and clinical practice. With the wealth of new longitudinal data, there is a critical need for visualizations for cancer researchers to explore and interpret temporal patterns not just in a single patient but across cohorts.
```
Comment (ngehlenborg) [inline _publications/harbig-2020-osf.md:19]:
```suggestion
Molecular profiling of patient tumors and liquid biopsies over time with next-generation sequencing technologies and new immuno-profile assays are becoming part of standard research and clinical practice. With the wealth of new longitudinal data, there is a critical need for visualizations for cancer researchers to explore and interpret temporal patterns not just in a single patient but across cohorts. 
```

File: _publications/harbig-2020-osf.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  published: "OSF Preprints b3n4u"
+---
+<b>Motivation:</b> Molecular profiling of patient tumors and liquid biopsies over time with next-generation sequencing technologies and new immuno-profile assays are becoming part of standard research and clinical practice. With the wealth of new longitudinal data, there is a critical need for visualizations for cancer researchers to explore and interpret temporal patterns not just in a single patient but across cohorts. 
+
+<b>Results:</b> To address this need we developed OncoThreads, a tool for the visualization of longitudinal clinical and cancer genomics and other molecular data in patient cohorts. The tool visualizes patient cohorts as temporal heatmaps and Sankey diagrams that support the interactive exploration and ranking of a wide range of clinical and molecular features. This allows analysts to discover temporal patterns in longitudinal data, such as the impact of mutations on response to a treatment, e.g. emergence of resistant clones. We demonstrate the functionality of OncoThreads using a cohort of 23 glioma patients sampled at 2-4 timepoints.
```
Comment (ngehlenborg) [inline _publications/harbig-2020-osf.md:21]:
```suggestion
To address this need we developed OncoThreads, a tool for the visualization of longitudinal clinical and cancer genomics and other molecular data in patient cohorts. The tool visualizes patient cohorts as temporal heatmaps and Sankey diagrams that support the interactive exploration and ranking of a wide range of clinical and molecular features. This allows analysts to discover temporal patterns in longitudinal data, such as the impact of mutations on response to a treatment, e.g. emergence of resistant clones. We demonstrate the functionality of OncoThreads using a cohort of 23 glioma patients sampled at 2-4 timepoints.
```

File: _publications/harbig-2020-osf.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  - qianwen-wang 
+  - nils-gehlenborg
+
+year: 2020
+type: preprint
+
+publisher: "https://osf.io/b3n4u/"
+cite:
+  authors: "Theresa Harbig, Sabrina Nusrat, Tali Mazor, Qianwen Wang, Alexander Thomson, Hans Bitter, Ethan Cerami, Nils Gehlenborg"
+  published: "OSF Preprints b3n4u"
```
Comment (ngehlenborg) [inline _publications/harbig-2020-osf.md:17]:
```suggestion
  published: "OSF Preprints"
```

File: _publications/harbig-2020-osf.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+title: "OncoThreads: Visualization of Large Scale Longitudinal Cancer Molecular Data"
+image: oncoThreads-osf
+
+members:
+  - theresa-harbig
+  - sabrina-nusrat
+  - qianwen-wang 
+  - nils-gehlenborg
+
+year: 2020
```
Comment (ngehlenborg) [inline _publications/harbig-2020-osf.md:11]:
```suggestion
year: 2021
```

File: _projects/oncothreads.md [hunk 5]
```diff
@@ -20,6 +21,9 @@ github_repositories:
 
 docker_repositories:
 
+publications:
+  - harbig-2020-osf
```
Comment (ngehlenborg) [inline _projects/oncothreads.md:25]:
```suggestion
  - harbig-2021-osf
```

**Review by ngehlenborg — CHANGES_REQUESTED** [review]
Thanks!

Made some minor edits. The year should be 2021 and I changed that in the file content. Can you make sure to rename the md file (which also contains 2020)?

**Review by ngehlenborg — APPROVED** [review]

**Comment by wangqianwen0418** [comment]
Thanks for the review and sorry for the silly mistake! Somehow forget it is already 2021 😳 
I have updated the pull request~

**Pull request merged**